src/mlui/enums/Adabound.py

```diff
@@ -0,0 +1,225 @@
+
+
+
+import tensorflow.compat.v2 as tf
+from keras import backend
+from keras.optimizers import Optimizer
+
+class AdaBound(Optimizer):
+    """
+    Adabound optimizer implementation 
+    # References
+        - [Adaptive Gradient Methods with Dynamic Bound of Learning Rate]
+          (https://openreview.net/forum?id=Bkg3g2R9FX)
+
+
+
```

Comment:
There should be one blank line between separate sections.

Reply:
solved